teamcity/flake8_plugin.py

```diff
@@ -1,74 +1,28 @@
-try:
-    import pycodestyle as pep8
-except ImportError:
-    import pep8
+from flake8.formatting import base
 
 import re
 
 from teamcity.messages import TeamcityServiceMessages
-from teamcity import __version__, is_running_under_teamcity
```

Comment:
It possible to follow http://flake8.pycqa.org/en/latest/plugin-development/cross-compatibility.html and retain compatibility with both versions. What do you think?